### PR TITLE
feat: show tooltip when hovering over 'Create new spec' button

### LINKED ISSUE #21858: Show tooltip when hovering over 'Create new spec' button within Specs panel in App
## **Summary**
It would be nice if the “create new spec” button had a similar tooltip like others in the UI. I’ll attach a screenshot in the thread.

Screen Shot 2022-04-21 at 12 02 06 PM

Screen Shot 2022-04-21 at 12 02 25 PM

┆Issue is synchronized with this [Jira Task](https://cypress-io.atlassian.net/browse/UNIFY-1644) by [Unito](https://www.unito.io)
┆Attachments: <a href="https://cypress-io.atlassian.net/rest/api/2/attachment/content/11853">Screen Shot 2022-04-21 at 12.02.06 PM.png</a> | <a href="https://cypress-io.atlassian.net/rest/api/2/attachment/content/11852">Screen Shot 2022-04-21 at 12.02.25 PM.png</a>
┆author: Adam Murray
┆epic: UI/UX Improvements
┆friendlyId: UNIFY-1644
┆priority: Low
┆sprint: Backlog
┆taskType: Task

### PR DESCRIPTION
**Pull request by gitstart**
- Closes https://github.com/cypress-io/cypress/issues/21858

### User facing changelog

Fix an issue where the user does not see a tooltip showing text `Create new spec` on the `+` button on hover.

### Additional details
Improve user accessibility making it clear the purpose of the `+` button on hover.

### Steps to test
- Hover Mouseover `+` Button. Should show tooltip `Create new spec`

#### Before
https://www.loom.com/share/dc1b7c5e47ef42e1bca42977dfc3cc21

#### After
https://www.loom.com/share/7a07b98860b346e1bb6f1c96c34dd74b

### PR Tasks

- [x] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

**Comment by gitstart** [comment]
Hi @marktnoonan Could you help to trigger CI build on this PR, thanks.

File: packages/frontend-shared/src/locales/en-US.json [hunk 1]
```diff
@@ -60,6 +60,7 @@
   "createSpec": {
     "newSpec": "New Spec",
     "createSpec": "Create Spec",
+    "createNewSpec": "Create new spec",
```
Comment (lmiller1990) [inline packages/frontend-shared/src/locales/en-US.json]:
```suggestion
    "createNewSpec": "Create New Spec",
```

**Review by lmiller1990 — COMMENTED** [review]
One comment. I'll run CI.

**Review by emilyrohrbough — CHANGES_REQUESTED** [review]
Why is tooltip needed in this scenario? The button's message is fine. This bloats the dom unnecessarily. If we need a tooltip we should provide a message that gives better insides into what "new spec means".

**Comment by gitstart** [comment]
Thanks for the review @emilyrohrbough,
The PR was created to resolve issue raised in #21858 with the assumption that a tooltip is desired as mentioned in issue. Do you propose that it might not be the case? or prefer a better tooltip message.
CC: @lmiller1990

**Comment by lmiller1990** [comment]
I kind of agree with @emilyrohrbough - I am not sure a button really needs a tooltip. I'm not sure how much value it adds.I think the original issue seems obsolete, we should have closed it before work was put into this.

I think we should close this. Sorry to waste your time/effort @gitstart - might be worth tagging someone before picking up something if there's any ambiguity.

**Pull request closed without merging**